.env.production

```
@@ -0,0 +1,14 @@
+NODE_ENV=production
+DATABASE_URL=postgresql://neondb_owner:npg_AGR4shun0lLM@ep-icy-bird-a9lng7lk-pooler.gwc.azure.neon.tech/neondb?sslmode=require&channel_binding=require
+CLOUDINARY_CLOUD_NAME=dqcjjscle
+CLOUDINARY_API_KEY=433843797839238
+CLOUDINARY_API_SECRET=YeXDJXHGCDtY9Cb0xSXuXNCvx_M
+
+AUDIENCE='http://localhost:3000/'
+ISSUER='https://bookingcrm.eu.auth0.com/'
+
+
+LIQPAY_PUBLIC_KEY=sandbox_i46500897238
+LIQPAY_PRIVATE_KEY=sandbox_CeMWt3CZlEKqxY0NOngJL2Iui0G8t7J426jFIwtg
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The LIQPAY_PRIVATE_KEY key should go before the LIQPAY_PUBLIC_KEY key

(UnorderedKey)